### PR TITLE
Multiline editor

### PR DESCRIPTION
**Pull request by Restioson**
[Behaviour demonstrated in a video](https://streamable.com/d481ed)

### CODE REVIEW

**Comment by Restioson** [comment]
Styling over to you @gegy1000, because I don't know what I'm doing!

File: client/gtk/src/screen/active.rs [hunk 1]
```diff
@@ -233,7 +292,8 @@ impl client::ClientUi for Ui {
         self.message_entry.set_can_focus(true);
         self.message_entry.set_editable(true);
 
-        self.message_entry.set_placeholder_text(Some("Send message..."));
+        // TODO
+        //self.message_entry.set_placeholder_text(Some("Send message..."));
```
Comment (Gegy) [inline client/gtk/src/screen/active.rs]: do we wanna fix these before merge?

**Comment by Gegy** [comment]
nice!! i'll work on styling this evening :)

File: client/gtk/src/screen/active.rs (hunk 1 above)
Comment (Restioson) [reply client/gtk/src/screen/active.rs]: yea

**Comment by Restioson** [comment]
Can we merge this?

**Review by Gegy — APPROVED** [review]

**Pull request merged**